{{ .ProjectSnake }}/BUILD.bazel

```diff
@@ -49,6 +49,7 @@ js_library(
 {{- end }}
 {{- if .Computed.python }}
 
+# aspect:exclude **/*.venv
```

Comment:
nit, add a comment to why this is ignored, the other directives come with an explanation.